package.json

```
@@ -14,14 +14,13 @@
     "clean": "pnpm turbo clean && pnpm storybook:clean && rimraf node_modules",
     "test": "pnpm turbo test",
     "test:ci": "pnpm turbo test:ci",
-    "storybook": "storybook dev -p ${STORYBOOK_PORT:-4479} -c storybook",
+    "storybook": "storybook dev -p ${STORYBOOK_PORT:-6006} -c storybook",
     "storybook:preview": "NODE_ENV=production pnpm storybook --docs",
     "storybook:clean": "rm -rf storybook/public",
-    "storybook:build": "storybook build -c storybook -o storybook/public",
-    "storybook:build:root": "pnpm turbo storybook:build",
-    "storybook:build:test": "pnpm turbo storybook:build -- --test",
-    "storybook:build:prod": "pnpm turbo storybook:build -- --docs",
-    "test:storybook": "test-storybook --config-dir storybook --skipTags='skip-test' --url http://127.0.0.1:4479",
+    "storybook:build": "storybook build -c storybook -o storybook/public --stats-json",
+    "storybook:build:test": "pnpm storybook:build -- --test",
+    "storybook:build:prod": "pnpm storybook:build -- --docs",
```

Pretty sure the `--` is a turbo thing :)
```suggestion
    "storybook:build:test": "pnpm storybook:build --test",
    "storybook:build:prod": "pnpm storybook:build --docs",
```